### PR TITLE
Expand netlify graph docs

### PR DESCRIPTION
**Pull request by KyleBlankRollins**
This PR updates the Netlify Graph docs to account for updated flows in the Netlify UI.

The README file serves as a hub for the other docs and resources. Only high-level information lives there. Detailed information goes in dedicated files.

When @localden is finished with the `gravity` sample and tutorial, I'll update further.

There are a number of `TODO` and `QUESTION` comment lines throughout. I'll remove these before the PR is merged.

### CODE REVIEW

File: features/graph/documentation/README.md [hunk 1]
```diff
@@ -1,38 +1,82 @@
...
+  1. [Link your local project directory to your Netlify site](authentication.md#link-your-local-project-directory-to-your-netlify-site)
+  2. [Connect to your first API or service](authentication.md#connect-to-your-first-api-or-service) in the Netlify UI.
+  3. [Start a local CLI session with Netlify Dev](authentication.md#start-a-local-cli-session-with-netlify-dev).
+  4. [Create a GraphQL query with Graph Explorer](authentication.md#create-a-graphql-query-with-graph-explorer).
+  5. [Generate a query handler](authentication.md#generate-a-query-handler).
+  6. [Use a query handler in your project](authentication.md#use-a-query-handler-in-your-project).
 
-## Enabling the feature
+## Queries, mutations, and subscriptions
+<!-- QUESTION: Are we going to address fragments? -->
```
Comment (KyleBlankRollins) [inline features/graph/documentation/README.md:49]: @localden, do we want to address fragments in the docs at all?

File: features/graph/documentation/README.md [hunk 2]
```diff
@@ -1,38 +1,82 @@
...
+<!-- QUESTION: Is my assumption that these are standard GraphQL correct? -->
```
Comment (KyleBlankRollins) [inline features/graph/documentation/README.md:50]: @localden, can you confirm/deny this?

File: features/graph/documentation/README.md [hunk 3]
```diff
@@ -1,38 +1,82 @@
...
+You use standard GraphQL queries, mutations, and subscriptions to interact with your Netlify Graph data.
 
-To enable Netlify Graph in your account, you need to use Netlify Labs, where you can click on **Enable** next to **Netlify Graph**.
+- **[Queries](https://graphql.org/learn/queries/).** Are how you get data.
+- **[Mutations](https://graphql.org/learn/queries/#mutations).** Are how you modify data. 
+- **[Subscriptions](https://www.onegraph.com/docs/subscriptions.html).** Let you receive data in response to an event.
 
-![Enabling Netlify Graph in Netlify Labs](../../../media/graph/enable-labs.gif)
+<!-- TODO: Possibly add examples, though I think linking to external sources is probably the best way to go. If we want to expand this section, we could have subsections for each type. If we want to spend the time on it and see value in it, we could also have a separate page dedicated to more in-depth explanation of these. -->
```
Comment (KyleBlankRollins) [inline features/graph/documentation/README.md:57]: I think we could expand on these and their differences in the future if we think there's value there. For now, I believe linking out to third-party info is better.

File: features/graph/documentation/README.md [hunk 4]
```diff
@@ -1,38 +1,82 @@
...
+## Access secrets with Netlify Graph
 
-![Top nav bar showing Netlify Graph](../../../media/graph/graph-top-nav.png)
+You can access secrets for all of the APIs and services you connect to Netlify Graph. Check out the [Authentication docs](/authentication.md#basic-secret-handling) for details.
+
+## Netlify Graph and Next.js
+
+If you are building a Next.js application, you can automatically generate Netlify Graph bindings by running the Netlify Graph commands inside your application folder.
+
+<!-- QUESTION: perhaps this is my unfamiliarity with Next.js, but what's the difference between the application root and the repository root? They're not the same? -->
```
Comment (KyleBlankRollins) [inline features/graph/documentation/README.md:67]: @localden, just want to make sure that the following note is still accurate.

File: features/graph/documentation/authentication.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
+# Graph Authentication
+
+Graph Authentication is the part of [Netlify Graph](README.md) that simplifies API authentication and token management. It removes the friction of re-writing authentication code to connect to third-party APIs.
+
+<!-- QUESTION: Are tokens still stored as env vars? -->
```
Comment (KyleBlankRollins) [inline features/graph/documentation/authentication.md:5]: @localden, I have this question in a few places. When I've authenticated in my testing, I haven't seen any new env vars. Are we handling tokens differently now?

File: features/graph/documentation/get-started.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+After generating a query handler and pulling it to your local project, you can access that handler in your project's code. Handlers are called just like other [Netlify Functions](https://docs.netlify.com/functions/build-with-javascript/).
+
+Queries are stored in your local project under `netlify/functions/YOUR_QUERY`, which means that you need to call their endpoints like this: `/.netlify/functions/YOUR_QUERY`. Note the preceding slash and period.
+
+You can test your query handlers locally by running `netlify dev --graph` at your project's root.
+
+To deploy the functions, commit the changes to your local project and push to your git remote origin. This starts a deploy of your Netlify site and its associated functions.
+
+<!-- QUESTION: Is this still relevant? -->
```
Comment (KyleBlankRollins) [inline features/graph/documentation/get-started.md:90]: @localden, the 2 following paragraphs don't seem relevant anymore. But maybe I'm missing something. Do users still need to pass an auth token with their calls to the generated handlers?

File: features/graph/documentation/README.md (hunk 1 above)
Comment (localden) [reply features/graph/documentation/README.md:49]: @KyleBlankRollins let's make this a fast-follow.

File: features/graph/documentation/README.md (hunk 2 above)
Comment (localden) [reply features/graph/documentation/README.md:50]: @KyleBlankRollins confirming - this is standard GraphQL

File: features/graph/documentation/README.md (hunk 4 above)
Comment (localden) [reply features/graph/documentation/README.md:67]: Yes, that should still be accurate @KyleBlankRollins

File: features/graph/documentation/authentication.md (hunk 5 above)
Comment (localden) [reply features/graph/documentation/authentication.md:5]: @KyleBlankRollins no, they are not - this is something that is now done through a Netlify-specific package. As in, you would not need to care about the Graph token.

File: features/graph/documentation/get-started.md (hunk 6 above)
Comment (localden) [reply features/graph/documentation/get-started.md:90]: @KyleBlankRollins no, they are still relevant. We don't auto-bind authentication to generated code.

**Review by localden — APPROVED** [review]

**Pull request merged**